.github/workflows/integration-test.yml

```diff
@@ -21,3 +21,5 @@ jobs:
           fixtures_file: 'fixtures/initial_data.json'
           openedx_imports_test_file_path: 'eox_core/edxapp_wrapper/tests/integration/test_backends.py'
           tutor_extra_commands_path: 'integration_tests/scripts/execute_tutor_extra_commands.sh'
+        continue-on-error: ${{ matrix.tutor_version == 'nightly' }}
```

Comment:
Is the `fail-fast` no longer needed? I see it in the cover letter, so I'm guessing it was dropped.

Reply:
@mariajgrimaldi yes `fail-fast ` is what actually works. This `continue-on-error` marks the tests in nightly as successful https://github.com/eduNEXT/eox-core/actions/runs/12036115374/job/33556601073?pr=308
I'll revert the change

Reply:
Good, thank you!